src/style.css

```diff
@@ -0,0 +1,140 @@
+body {
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  background: #e5e5e5;
+}
+
+.conteiner {
+  width: 200px;
+  height: 408px;
+}
+
+.Product {
+  height: 406px;
+  width: 198px;
+  background: #fff;
```

Comment:
Move all repeatable colors to the variables.

[More info.](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties)